SEPACommon.pas

```diff
@@ -245,6 +245,9 @@ function CharIsSEPAWhitelisted(const c: Char): Boolean;
             (c = '''') or (c = ':') or (c = '?') or
             (c = ',') or (c = '-') or (c = ' ') or
             (c = '(') or (c = '+') or (c = '.') or
+
+            (c = ';') or   // We allow the ; char so we can use XML-Escapes like &amp;
```

Comment:
I'm still not sure this is the right way. On the other hand, it gives the flexibility to use escaping when the own financial institution accepts this.
In the future: please prepare separate branches/pull requests for different features.